### PR TITLE
Budgs Bloat Clothes, Boes and Shoots

### PR DESCRIPTION
**Pull request by BulletBudgie**
A collection of colourable boots added to the loadout:
+boot selection (colourable) / shoe tab
+toeless boot selection (colourable) / xenowear tab
+thigh-high sneakers / shoe selection (colourable)

All onmob sprites by me
Most item sprites borrowed and altered from existing sprites

### CODE REVIEW

**Comment by BulletBudgie** [comment]
!review

**Comment by alsoandanswer** [comment]
Hard no on the high heel boots.

**Comment by BulletBudgie** [comment]
> Hard no on the high heel boots.

can you elaborate? there are three sized of heeled boots, i am happy to remove the ones you feel are not acceptable

**Comment by alsoandanswer** [comment]
> can you elaborate? there are three sized of heeled boots, i am happy to remove the ones you feel are not acceptable

All of the high heel boots, unfortunately. The original PR that added the thigh high and knee high jackboots were already pushing the limit of believability, and I do not think high-heeled variants would be acceptable. The final judgement rests on the maintainers, however.

**Comment by NonQueueingMatt** [comment]
Have to agree with the removal of high heel boots I think

**Comment by BulletBudgie** [comment]
Removed the high-heeled variants of the boots - I did keep the low-heeled ankle boots (/obj/item/clothing/shoes/ankleboots) as I don't think they push the realms of whats believable.

File: code/modules/client/preference_setup/loadout/items/xeno/tajara.dm [hunk 1]
```diff
@@ -42,6 +50,17 @@
 	boots["adhomian boots"] = /obj/item/clothing/shoes/workboots/tajara/adhomian_boots
 	gear_tweaks += new /datum/gear_tweak/path(boots)
 
+/datum/gear/shoes/tajara/recolourable_boots/New()
```
Comment (Geevies) [inline code/modules/client/preference_setup/loadout/items/xeno/tajara.dm:53]: move this to be beneath the main definition, you could move the one above as well

File: html/changelogs/budg-bloatclothes-shoes-and-boots.yml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+#   config
+#     - (makes a change to the config files)
+#   admin
+#     - (makes changes to administrator tools)
+#   server
+#     - (miscellaneous changes to server)
+#################################
+
+# Your name.
+author: ChangeMe
```
Comment (Geevies) [inline html/changelogs/budg-bloatclothes-shoes-and-boots.yml:47]: changeme

File: html/changelogs/budg-bloatclothes-shoes-and-boots.yml [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, this gets changed to [] after reading.  Just remove the brackets when you add new shit.
+# Please surround your changes in  double quotes ("). It works without them, but if you use certain characters it screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Added a changelog editing system that should cause fewer conflicts and more accurate timestamps."
```
Comment (Geevies) [inline html/changelogs/budg-bloatclothes-shoes-and-boots.yml:58]: changelog

**Review by Geevies — CHANGES_REQUESTED** [review]

**Comment by NonQueueingMatt** [comment]
yep, ankle boots are fine, PR is good to go from our perspective

**Comment by Greenjoe12345** [comment]
Don't really agree with the ruling on high heel boots, those are literally a thing irl, so it doesn't seem unreasonable to have?

File: html/changelogs/budg-bloatclothes-shoes-and-boots.yml (hunk 2 above)
Comment (BulletBudgie) [reply html/changelogs/budg-bloatclothes-shoes-and-boots.yml]: done

File: code/modules/client/preference_setup/loadout/items/xeno/tajara.dm (hunk 1 above)
Comment (BulletBudgie) [reply code/modules/client/preference_setup/loadout/items/xeno/tajara.dm:53]: This was added before I realized that auto-adapt sprites (the ones that taj uses) work on items that aren't in the xenowear tab; this code is functionless and I must've forgotten to remove it, I'll do that now.

I'm not sure what you mean by moving the list to the 'main definition' I could do that to the existing list if you need me to however I'm planning on removing my vestigial list.

**Review by Arrow768 — APPROVED** [review]

**Review by Geevies — APPROVED** [review]

**Pull request merged**